packages/terra-collapsible-menu-view/src/terra-dev-site/doc/collapsible-menu-view/CollapsibleMenuView.1.doc.mdx

```diff
@@ -0,0 +1,70 @@
+import { Badge } from 'terra-collapsible-menu-view/package.json?dev-site-package';
+
+import CollapsibleMenuViewProps from 'terra-collapsible-menu-view/src/CollapsibleMenuView?dev-site-props-table';
```

Comment:
This import is causing the build/compile stage to fail. It may be related to missing the underscore on the filename.

```
terra-framework/packages/terra-collapsible-menu-view/src/_CollapsibleMenu.jsx
```

Reply:
Changing the Prop types to use the values directly instead of Object.keys fixed the build failure.